odometry/package.xml

```diff
@@ -0,0 +1,67 @@
+<package format="2">
+  <name>odometry</name>
+  <version>0.0.0</version>
```

Comment:
change to version 0.1.0  - we should be following the pattern of [semantic versioning](https://semver.org/)